emily/handler/tests/integration/deposit.rs

```diff
@@ -1013,29 +1008,58 @@ async fn update_deposits_is_forbidden(
         })));
     }
 
-    let response = apis::deposit_api::update_deposits(
-        &user_configuration,
-        UpdateDepositsRequestBody {
-            deposits: vec![DepositUpdate {
-                bitcoin_tx_output_index: bitcoin_tx_output_index,
-                bitcoin_txid: bitcoin_txid.clone().into(),
-                fulfillment,
-                status: new_status,
-                status_message: "foo".into(),
-            }],
-        },
-    )
-    .await;
+    #[derive(Debug)]
+    enum DepositUpdateError {
+        Signer(apis::Error<apis::deposit_api::UpdateDepositsSignerError>),
```

Comment:
QQ: Do these errors actually have different content, or could it have used the same error type for both (since these endpoints are the same in spirit)?

Reply:
Yeah, generated code produce different types of errors (actually identical inside), and it don't allow to compile them without such a hack. Probably it is not the best hack to do, but some hack is needed. Or, we can divide this test in two, as Francesco suggested

Reply:
I agreed with Francesco and deleted this enums at all: [89d24c4](https://github.com/stacks-sbtc/sbtc/pull/1586/commits/89d24c4e09852299089583ff891fbd22e9e1f283)